packages/tools/test/getVolumeId_test.js

```diff
@@ -0,0 +1,15 @@
+import { getVolumeId } from '../../tools/src/utilities/getVolumeId';
```

Comment:
This can be ../src/utilities/getVolumeId

Reply:
Your suggestion doesn't work.